src/vs/platform/screenshot/browser/screenshot.ts

```diff
@@ -0,0 +1,139 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
I see a monaco checks error. We could move this file into workbench/contrib/chat in the meantime until it's actually used elsewhere? Maybe ESM handles tree shaking of this now in the monaco package, but I'm not 100% sure.

Reply:
hm, tons of import violations when I move it there. i'll see where else it could go

Reply:
actually, monaco check seems fine now?